src/main/java/core/basesyntax/Fruits.java

```diff
@@ -0,0 +1,41 @@
+package core.basesyntax;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public abstract class Fruits {
+    private Map<String, Integer> fruitQuantities;
+
+    public int getFruitQuantities(String fruitType) {
+        return fruitQuantities.get(fruitType);

```

Comment:
The method `getFruitQuantities` may throw a `NullPointerException` if the `fruitType` does not exist in the map. Consider using `fruitQuantities.getOrDefault(fruitType, 0)` to return a default value of 0 if the fruit type is not present.